src/System.Security.Cryptography.Pkcs/tests/Pkcs9AttributeWindowsFileTimeTests.cs

```diff
@@ -0,0 +1,28 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using Xunit;
+
+using Test.Cryptography;
+
+namespace System.Security.Cryptography.Pkcs.Tests
+{
+    public static class Pkcs9AttributeWindowsFileTimeTests
+    {
+        [Fact]
+        public static void InputDateTimeAsWindowsFileTimeBefore1601()
+        {
+            DateTime dt = new DateTime (1600, 12, 31, 11, 59, 59, DateTimeKind.Utc);
+            Assert.ThrowsAny<CryptographicException>(() => new Pkcs9SigningTime(dt));
+        }
+
+        [Fact]
+        public static void InputDateTimeAsWindowsFileTimeMinValue()
```

Comment:
Since the input is DateTime.MinValue I'm not sure what FILETIME has to do with it.

Reply:
well, this is just another guaranteed case when an input date is less then minimal bound of file time.

Reply:
I'm not saying it's a bad test case, my complaint is with the name.  `Pkcs9SigningTime_DateTimeMinValue` seems pretty well descriptive enough, assuming it gets put back in Pkcs9AttributeTests.cs, or `InputDateTimeMinValue` in the new file here.